src/modules/requestAPIMovies.js

```diff
@@ -0,0 +1,13 @@
+import renderMovies from './renderMovies.js';
+
+const getAPIMovies = async () => {
+  const apiURL = 'https://api.tvmaze.com/shows';
+  const request = new Request(apiURL);
+  const response = await fetch(request);
+  const textMovieData = await response.text();
```

Comment:
Kindly, fetching data in JSON file format makes it easier to use with JavaScript